src/google-contetnt-provider.ts

```diff
@@ -0,0 +1,232 @@
+import config from '../config.json';
```

Comment:
Instead of reading these values from the JSON configuration file, we should provide them as parameters in the constructor for the content provider.

Reply:
service account credentials as well?